File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if (len(nums) == 0):
            return 0
        nums.sort();
        current_sequence = 1
        longest_sequence = 1
        i = 0
        for i in range(len(nums) - 1):
            if (nums[i + 1] == nums[i]):
                continue;
            if (nums[i + 1] == nums[i] + 1):
                current_sequence+=1
                if (current_sequence > longest_sequence):
                    longest_sequence = current_sequence
            else:
                current_sequence = 1
        return longest_sequence

File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] solution = new int[2];
        boolean found = false;
        for (int i = 0; i < nums.length; i++) {
            if (found == true) {
                break;
            }
            for (int j = i+1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    solution[0] = i;
                    solution[1] = j;
                    found = true;
                    break;
                }
                else {
                    continue;
                }
            }
        }
        return solution;
    }
}
